pages/api/slack/appSubmit.ts

```diff
@@ -0,0 +1,33 @@
+import { App, LogLevel } from "@slack/bolt";
+import dotenv from "dotenv";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+dotenv.config();
+
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  signingSecret: process.env.SIGNING_SECRET,
```

Comment:
did u test if these are null?

Reply:
It will fail but doesn't really matter because the function is only called on prod and idc if it errors on staging/local. And if it does it doesn't take anything else down with it